### ISSUE TITLE
Can't coerce with mappings anymore

### ISSUE THREAD

**Issue by martskins** [tpope/vim-abolish#72]
After updating my plugins today (`PlugUpdate`) I can't change casing via the coerce mappings anymore. Any `cr<anything>` mapping will just move the cursor to the end of the word and leave the casing unchanged.

The output of `:verbose nmap crs` seems fine to me: `n cr  <Plug>(abolish-coerce)iw`

Did something change lately?

**Comment by martskins** [comment]
I think this might be solved by #71

**Comment by TheRealest** [comment]
I am having this same issue and took a peek at the latest commits -- it looks like this is the culprit: https://github.com/tpope/vim-abolish/commit/62a593e9fcccc33afc63cc94e5d08da7c52a4fab#diff-832897d93a1d6a93e135c87c934cafd8R629

```
nnoremap <expr> <Plug>(abolish-coerce) <SID>coerce(nr2char(getchar())).'iw'
nnoremap <expr> <Plug>Coerce <SID>coerce(nr2char(getchar())).'iw'

...

nmap cr  <Plug>Coerce	  nmap cr  <Plug>(abolish-coerce)iw
```
The plugin command mappings already include the text object `iw`, but the `cr` normal command mapping is adding the same text object again at the end. In the meantime while this is being fixed you can just remap `cr` to the plugin command without the `iw` at the end.
